### PR TITLE
JOSS paper review responses

### PR DESCRIPTION
**Pull request by hkaiser**
In response to https://github.com/openjournals/joss-reviews/issues/2352#issuecomment-666381389:

- Adding link to governance document to CONTRIBUTING.md
- Adding notes that we welcome pull requests 
- Referring to governance document from HPX docs

### CODE REVIEW

File: docs/sphinx/contributing/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    Copyright (C) 2020 Hartmut Kaiser
+
+    SPDX-License-Identifier: BSL-1.0
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+.. _contributing:
+
+===================
+Contributing to HPX
```
Comment (msimberg) [inline docs/sphinx/contributing/contributing.rst:11]:
```suggestion
Contributing to |hpx|
```

File: docs/sphinx/contributing/contributing.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+===================
+
+The main source of information to understand the process of how to contribute
+to HPX can be found in
```
Comment (msimberg) [inline docs/sphinx/contributing/contributing.rst:15]:
```suggestion
to |hpx| can be found in
```

File: docs/sphinx/contributing/governance.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    Copyright (C) 2020 Hartmut Kaiser
+
+    SPDX-License-Identifier: BSL-1.0
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+.. _governance:
+
+====================
+HPX Governance model
```
Comment (msimberg) [inline docs/sphinx/contributing/governance.rst:11]:
```suggestion
|hpx| governance model
```

File: docs/sphinx/contributing/governance.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+====================
+
+The HPX project is a meritocratic, consensus-based community project. Anyone
```
Comment (msimberg) [inline docs/sphinx/contributing/governance.rst:14]:
```suggestion
The |hpx| project is a meritocratic, consensus-based community project. Anyone
```

**Comment by hkaiser** [comment]
@msimberg thanks, I have applied your suggestions.

**Pull request merged**